instrumentation/spring/spring-webmvc-3.1/library/src/main/java/io/opentelemetry/instrumentation/spring/webmvc/WebMvcTracingFilter.java

```diff
@@ -30,8 +31,8 @@ public WebMvcTracingFilter(Tracer tracer) {
   public void doFilterInternal(
       HttpServletRequest request, HttpServletResponse response, FilterChain filterChain)
       throws ServletException, IOException {
-    Span serverSpan = tracer.startSpan(request, request, FILTER_CLASS + "." + FILTER_METHOD);
-
+    Context ctx = tracer.startSpan(request, request, FILTER_CLASS + "." + FILTER_METHOD);
+    Span serverSpan = Span.fromContext(ctx);
```

Comment:
I think we can optimise this pattern, without taking span from context and then putting it back in `startScope`. But as @marcingrzejszczak  really wants it now, let's do it in follow up